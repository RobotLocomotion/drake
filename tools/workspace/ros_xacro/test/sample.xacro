<?xml version="1.0" ?>
<root xmlns:xacro="http://www.ros.org/wiki/xacro">
  <xacro:property name="diameter" value="2.0"/>
  <sphere radius="${diameter/2}"/>
</root>
