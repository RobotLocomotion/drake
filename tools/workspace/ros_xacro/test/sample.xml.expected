<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from tools/workspace/ros_xacro/test/sample.xacro | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<root>
  <sphere radius="1.0"/>
</root>
